www/base/src/app/builders/builds/build.controller.coffee

```diff
@@ -72,7 +72,7 @@ class Build extends Controller
         data = dataService.open($scope)
         data.getBuilders(builderid).then (builders) ->
             $scope.builder = builder = builders[0]
-            builder.getBuilds(number__lt: buildnumber + 2, limit: 3, order: '-number').then (builds) ->
+            builder.loadBuilds(number__lt: buildnumber + 2, limit: 3, order: '-number').then (builds) ->
```

Comment:
mmh indeed, this does not look right, we should not rely on the fact that we store a reference of the the element to stop the listening.

I think each wrapped element should have a reference on the dataService.open handle, so that it can registers its sub registrations properly to be able to release them afterwards